Issue #122 update version numbers

bikegeek opened the pull request:
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
verified that GHA does not produce warnings

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
 verify that version numbers updated for compare_db.yml and main.yactionsml for (wherever applicable):
  - Update version of actions/checkout to v3
  - Update version of actions/setup-python to v4
  - Update version of actions/upload-artifact to v3
  - Update version of actions/download-artifact to v3
  - Update version of actions/github-script to v6


 

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[NA]**

- [x] Do these changes include sufficient testing updates? **[NA]**

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **[At earliest convenience, to meet beta4 release]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR definition above.
- [ ] Ensure the PR title matches the feature or bugfix branch name.
- [ ] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

bikegeek committed:
Issue #122 update version numbers

venitahagerty commented:
Looks like line 34 also needs to be changed in compare_db.yml for the second checkout.

bikegeek committed:
Issue #122 missed one version to update

bikegeek committed:
Issue #122 update ubuntu to 20.04

bikegeek committed:
return ubuntu to 18.04, this will be unsupported in 2023

bikegeek commented:
updated the remaining version, but there is a warning with the ubuntu-18.04 version.  I updated the version and get an error message about the sql syntax.  I created a new Github issue to deal with this: https://github.com/dtcenter/METdataio/issues/128

venitahagerty reviewed (approved):
Looks good to me.
